### PR TITLE
Aligned the contents of the navbar in the center

### LINKED ISSUE #309: aligning the contents of navbar in the center
## **Describe the bug**

- The links in the small and medium screen are on the extreme right of the page which does not look good. 

## **Possible solution**

-The links can be aligned in the center.

## **Screenshots**

- **BEFORE**
Screenshot (9)

**AFTER**
Screenshot (10)

### PR DESCRIPTION
**Pull request by 123mansha**
# Description

Please include a summary of the change and which issue is fixed.
- Aligned the contents of the navbar in the center.
- Changed the transparent background to solid background 
Fixes #309

## List any dependencies that are required for this change 
no dependencies

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
Screenshot (9)

    - After :

Screenshot (12)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by ravikr126 — APPROVED** [review]

File: css/style.css [hunk 1]
```diff
@@ -2,258 +2,259 @@
...
-    font-weight: bold;
+  font-size: 18px;
+  font-weight: bold;
 }
 
 .nav-col {
-    background-color: #0e2431;
-    opacity: 0.8;
+  background-color: #0e2431;
+  /*opacity: 0.8;*/
```
Comment (harikesh409) [inline css/style.css:97]: Remove comment if not needed.

File: css/style.css [hunk 2]
```diff
@@ -2,258 +2,259 @@
...
 }
 
-.top{
-    background: transparent;
-}
+/*.top {
```
Comment (harikesh409) [inline css/style.css:100]: Remove comment if not needed.

File: css/style.css [hunk 3]
```diff
@@ -2,258 +2,259 @@
...
 
 .navbar-nav .nav-link {
-    color: #fff;
+  color: #fff;
 }
 
 .navbar-nav {
-    text-align: right;
-    
+  /*text-align: right;*/
```
Comment (harikesh409) [inline css/style.css:129]: Remove comment if not needed.

**Review by harikesh409 — CHANGES_REQUESTED** [review]
image
The content under the more is aligned to left it looks better if it is also aligned to center.

**Comment by 123mansha** [comment]
Okk, I will do

**Review by ravikr126 — APPROVED** [review]

**Pull request merged**